examples/07_end_to_end/seismic_waveform.ipynb

```diff
@@ -0,0 +1,1322 @@
+   "metadata": {},
+   "source": [
+    "The \"id\" of the Trace object is composed of the network id (\"BW\"), the station id (\"RJOB\") and the channel (\"EHZ\"). The channel names are composed of the band code (i.e. frequency, where \"E\" is \"Extremely High Frequency\"), the instrument code (\"H\" is \"High Gain Seismometer\") and the orientation code (i.e. \"Z\" is vertical movement). You can read more about channel naming [here](https://ds.iris.edu/ds/nodes/dmc/data/formats/seed-channel-naming/).\n",
+    "\n",
+    "The trace has two main attributes, \"stats\" which contains metadata such as the start/end time, sampling frequency, etc and \"data\" which is a numpy array of the raw measurements, at equally sampled intervals in the time range."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "a2770d97-aa2e-451a-91e0-7e160194d87e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "         network: BW\n",
+       "         station: RJOB\n",
+       "        location: \n",
+       "         channel: EHZ\n",
+       "       starttime: 2009-08-24T00:20:03.000000Z\n",
+       "         endtime: 2009-08-24T00:20:32.990000Z\n",
+       "   sampling_rate: 100.0\n",
+       "           delta: 0.01\n",
+       "            npts: 3000\n",
+       "           calib: 1.0\n",
+       "    back_azimuth: 100.0\n",
+       "     inclination: 30.0\n",
+       "        response: Channel Response\n",
+       "\tFrom M/S (Velocity in Meters Per Second) to COUNTS (Digital Counts)\n",
+       "\tOverall Sensitivity: 2.5168e+09 defined at 0.020 Hz\n",
+       "\t4 stages:\n",
+       "\t\tStage 1: PolesZerosResponseStage from M/S to V, gain: 1500\n",
+       "\t\tStage 2: CoefficientsTypeResponseStage from V to COUNTS, gain: 1.67785e+06\n",
+       "\t\tStage 3: FIRResponseStage from COUNTS to COUNTS, gain: 1\n",
+       "\t\tStage 4: FIRResponseStage from COUNTS to COUNTS, gain: 1"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tr.stats"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "0f8fde18-2112-4e60-bf56-a377ae1f0db3",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([0.        , 0.00694644, 0.07597424, ..., 1.93449584, 0.98196204,\n",
+       "       0.44196924])"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tr.data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1faba8a2-25ce-4610-b8ab-9635d7b9ea80",
+   "metadata": {},
+   "source": [
+    "## Simulation\n",
+    "In real-time, trace data will be sent in batches of observations, so we can use csp to simulate this by writing a node that divides a stream into a time series of pieces based on an interval, and assuming a certain transmission lag"
```

Comment:
I found this sentence confusing. I would change:
"divides a stream into a time series of pieces based on an interval, and assuming a certain transmission lag" 
to 
"divides a stream into fixed sized chunks for each interval. We can also simulate a lag in the transmitted data, as we would see in realtime event processing."